tests/test_cli.py

```diff
@@ -0,0 +1,133 @@
+"""Test argument parsing"""
+
+import os
+import logging
+from pathlib import Path
+
+import click
+import pytest
+from click.testing import CliRunner
+
+from rpmlb import LOG
+from rpmlb.cli import run
+
+
+# Module-level CLI runner
+runner = CliRunner()
+
+
+def test_parse_argv_no_options(tmpdir):
+    """Tests proper default values of the CLI"""
+
+    RECIPE_FILE = tmpdir.join('ror.yml')
+    RECIPE_NAME = 'rh-ror50'
+
+    # Prepare environment
+    RECIPE_FILE.write('')
+
+    # Parse the arguments
+    with tmpdir.as_cwd():
+        argv = list(map(str, [RECIPE_FILE, RECIPE_NAME]))
+        args = run.make_context('rpmlb', argv).params
+
+    current_dir = os.path.abspath(os.getcwd())
+
+    assert args['recipe_file'] == str(RECIPE_FILE), 'Wrong recipe file path'
+    assert args['recipe_name'] == str(RECIPE_NAME), 'Wrong recipe name'
+    assert args['build'] == 'dummy', 'Wrong builder name'
+    assert args['download'] == 'none', 'Wrong downloader name'
+    assert args['branch'] is None, 'Superfluous branch'
+    assert args['source_directory'] == current_dir
+
+
+@runner.isolated_filesystem()
+def test_log_verbosity():
+    """Ensure that the verbosity is set properly on."""
+
+    # Initial state – if the test fails here, the app has changed
+    # and the test needs to be adjusted.
+    assert LOG.getEffectiveLevel() == logging.INFO
+
+    recipe = Path('recipe.yml')
+    recipe.touch()
+
+    runner.invoke(run, ['--verbose', str(recipe), 'test'])
+    assert LOG.getEffectiveLevel() == logging.DEBUG
+
+    runner.invoke(run, [str(recipe), 'test'])
+    assert LOG.getEffectiveLevel() == logging.INFO
```

Comment:
Could you split this to a parametric test with verbose set tot True and False? Testing just one thing in one test is better. (Let me know if you need help with that.)